configure.ac

```diff
@@ -192,6 +192,9 @@ AC_ARG_ENABLE(trivial-httpd-cmdline,
   [Continue to support "ostree trivial-httpd" [default=no]])],,
   enable_trivial_httpd_cmdline=no)
 AM_CONDITIONAL(BUILDOPT_TRIVIAL_HTTPD, test x$enable_trivial_httpd_cmdline = xyes)
+AS_IF([test x$with_soup = xno && test x$enable_trivial_httpd_cmdline = xyes], [
+  AC_MSG_WARN([trivial-httpd enabled, but libsoup is not; libsoup is needed for trivial-httpd])
```

Comment:
I think this should probably be fatal; it's too easy to miss warnings in the autoconf output spew.